scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -182,7 +206,15 @@ def build_graph(df, dataset):
  
     df = muon_selections.veto_electrons(df)
     df = muon_selections.apply_met_filters(df)
-    df = muon_selections.apply_triggermatching_muon(df, dataset, "goodMuons_eta0", "goodMuons_phi0")
+    if not (args.vqt3dsmoothing and (args.vqtTestStep < 2)) :
+        df = muon_selections.apply_triggermatching_muon(df, dataset, "goodMuons_eta0", "goodMuons_phi0")
+
+    if (args.vqt3dsmoothing) :
+        if dataset.group in ["Top", "Diboson"]:
```

Comment:
In utilities/common.py we have background_MCprocs which includes Top, Diboson, and also QCD, exactly for these kinds of hack, better to use it here so that if we ever add more samples to be treated similarly we can just edit that array.